lib/project_types/script/layers/infrastructure/languages/assemblyscript_project_creator.rb

```diff
@@ -11,23 +11,73 @@ class AssemblyScriptProjectCreator
           property! :script_name, accepts: String
           property! :path_to_project, accepts: String
 
-          BOOTSTRAP = "npx --no-install shopify-scripts-toolchain-as bootstrap --from %{extension_point} --dest %{base}"
           BUILD = "shopify-scripts-toolchain-as build --src src/shopify_main.ts " \
           "--binary build/script.wasm --metadata build/metadata.json"
-          MIN_NODE_VERSION = "14.5.0"
+          MIN_NODE_VERSION = "14.5.0" # kept because task_runner uses this
           ASC_ARGS = "-- --lib node_modules --optimize --use Date="
 
+          EP_REPO = "https://github.com/Shopify/extension-points.git"
```

Comment:
Just want to note that this will break for beta partners since the repo isn't public yet

Reply:
Adam is working on designing the new repos, so hopefully soon.  Also, nitpick: should it be `API_REPO`?  Also, what do you think about taking the path to the project folder to bootstrap from as a parameter to `create`, then, we can potentially bootstrap from an Example Script, or other repos as well.